bindings/CMakeLists.txt

```diff
@@ -1,17 +0,0 @@
-#  Copyright 2021 The casbin Authors. All Rights Reserved.
```

Comment:
Why remove the bindings?

Reply:
Pybindings are now shifted to `/pycasbin`

Reply:
I think if we have double copy of this code, we must have to keep`pycasbin` and `bindings/pycasbin` consistent. I think it's unnecessary extra work. If we want to binding other language, we can `mkdir` like `pycasbin` to store its code.